client/src/components/TreeDropdownField/TreeDropdownField.scss

```diff
@@ -96,14 +101,22 @@
   }
 }
 
-.is-focused:not(.is-open)>.Select-control,
-.is-open>.Select-control,
-.Select-menu-outer {
+.Select-arrow-zone {
+  vertical-align: initial;
+}
+
+.Select-control {
+  border-color: $gray-200;
   box-shadow: none;
+}
+
+.Select.is-focused:not(.is-open) > .Select-control,
+.is-open > .Select-control {
   border-color: $brand-primary;
 }
```

Comment:
![image](https://user-images.githubusercontent.com/541886/116501168-3fc88000-a904-11eb-830d-fcdb522acb8d.png)

Need to move box shadow to `.Select.is-focused:not(.is-open) > .Select-control`.

Reply:
Have added `box-shadow: none` so that the box shadows doesn't show

The box shadows comes from react-select - the DSM linked in the parent issue has no box-shadows so this seems correct https://projects.invisionapp.com/dsm/silver-stripe/silver-stripe/asset/components/5acd939108f7520011a65e5c